### DISCUSSION TITLE
How does a browser program written with cefsharp support Youdao word segmentation

### DISCUSSION THREAD

**Discussion by Chenpppeng** [cefsharp/CefSharp#4544] [General]
This is how Youdao Dictionary looks in Google Chrome, with a red translation icon
![image](https://github.com/cefsharp/CefSharp/assets/58911110/4aba2724-013d-421d-a0cf-288162a5a110)
This is a browser program that I compiled myself by referencing cefsharp and it shows a translation icon in the address bar
![image](https://github.com/cefsharp/CefSharp/assets/58911110/6e849843-3ac0-43ea-9ecd-eb41f376c171)
However, Youdao Translation icon does not appear in web pages opened through the cefsharp program that you compiled
![image](https://github.com/cefsharp/CefSharp/assets/58911110/a182d06e-7e73-40c3-85b5-64d419bb0416)

[Youdao Download link](https://fanyi.youdao.com/download-Windows?keyfrom=fanyiweb_navigation)

**Comment by amaitland** [comment]
> Youdao word segmentation

Never heard of it. Know nothing of how it's implemented. 

You'll need to do some research to determine how it works and if [CEF](https://github.com/chromiumembedded/cef) supports it.

Please read the first few paragraphs of https://github.com/cefsharp/CefSharp/wiki/General-Usage
